docker/release/README.md

```diff
@@ -0,0 +1,105 @@
+## OpenSearch/Dashboards Docker Build README
+
+### Summary
+We support building OpenSearch/Dashboards docker images with [multi-CPU architecture support](https://docs.docker.com/desktop/multi-arch/).
+We also support old school single CPU architecture build as well.
```

Comment:
"also" and "as well" -> ... replace this whole thing with 

```
We support building OpenSearch and OpenSearch Dashboards docker [multi-CPU](https://docs.docker.com/desktop/multi-arch/) and single CPU architectures.
```

Reply:
thanks.